### PR TITLE
test

### PR DESCRIPTION
**Pull request by JoaoNacho-SG**
added something to readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,4 @@
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
 Its me Mario!
+Hello Mario!
```
Comment (catlaranjeiro) [inline README.md]: This is not the best option. Please remove

**Review by catlaranjeiro — CHANGES_REQUESTED** [review]
Please address the commented changes request

File: README.md (hunk 1 above)
Comment (JoaoNacho-SG) [reply README.md]: removed as requested

**Review by catlaranjeiro — APPROVED** [review]
all good 🚀

**Pull request merged**